templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,15 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                {{ manufacturer.cars.count }}
+              </td>
+              <td>
+                <a class="btn btn-info" href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a class="btn btn-danger" href="{% url "taxi:manufacturer-delete" manufacturer.id %}">Delete</a>
```

Comment:
The URL tag uses double quotes, which can cause issues. Use single quotes for the URL tag: `{% url 'taxi:manufacturer-delete' manufacturer.id %}`.